xmtp_api_grpc/src/grpc_api_helper.rs

```diff
@@ -31,15 +31,18 @@ use xmtp_proto::{
 
 async fn create_tls_channel(address: String) -> Result<Channel, Error> {
     let channel = Channel::from_shared(address)
```

Comment:
Now I'm not sure these changes are going to fix it cause it looks like the transport error is only thrown from this call

`https://github.com/hyperium/tonic/pull/920/files#diff-2dc4a5ebbcd9a8a198e55baa6958f271c1df257a9e3d6ae9c70295a4df7773deR77`